src/css/style.css

```diff
@@ -26,141 +51,217 @@ header #timer.running {
   color: lightgreen;
 }
 header #timer.gameover {
-  color: red;
+  color: var(--danger);
 }
 
 ul {
   list-style-type: none;
   margin: 0;
   padding: 0;
+  display: flex;
+  justify-content: space-around;
 }
 
-li {
-  display: inline-block;
+header {
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+.header_button {
+  background: var(--primary-gradient);
+  width: 110px;
+  padding: 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: var(--s-border-radius);
   color: white;
-  opacity: 0.9;
-  font-size: 20px;
-  border: 1px solid white;
-  padding: 15px;
-  width: 50px;
-  text-align: center;
-  margin-right: 15px;
-  border-radius: 6px;
+  border: 0;
+  font-size: 0.9rem;
+}
+
+.header_button_start {
+  box-shadow: var(--drop-shadow);
 }
 
-li:hover {
+.header_button:hover {
   cursor: pointer;
-  background-color: #28608a;
+  background-color: var(--light-grey);
 }
 
-li:active {
-  background-color: #225277;
+.header_button:active {
+  background-color: var(--primary);
 }
 
-section#game_canvas {
-  background-color: #347bb2;
-  flex-grow: 9;
-  display: flex;
-  justify-content: center;
-  align-items: center;
+.header_button--danger {
+  color: var(--danger);
+  background: none;
+}
+
+.header_button--danger:hover {
+  background: var(--danger);
+  color: var(--white);
+}
+
+.header_button:disabled {
+  background: var(--light-grey);
+  box-shadow: none;
+  transition: all 1s;
 }
 
 .rhyme_card {
-  background-color: white;
-  width: 35%;
-  height: 275px;
-  border-radius: 4px;
+  background: var(--primary-gradient);
+  height: 350px;
+  border-radius: var(--l-border-radius);
   display: flex;
   flex-direction: column;
   justify-content: space-between;
   align-items: center;
   transition: all 1s;
+  box-shadow: var(--drop-shadow);
 }
 
 .rhyme_card.correct {
-  animation: shake 0.72s cubic-bezier(0.36, 0.07, 0.19, 0.97) both;
   transform: translate3d(0, 0, 0);
   background-color: lightgreen;
 }
 
 .rhyme_card.incorrect {
   animation: shake 0.52s cubic-bezier(0.36, 0.07, 0.19, 0.97) both;
   transform: translate3d(0, 0, 0);
-  background-color: red;
-}
-
-@keyframes shake {
-  10%,
-  90% {
-    transform: translate3d(-1px, -2px, 0);
-  }
-  20%,
-  80% {
-    transform: translate3d(2px, 3px, 0);
-  }
-  30%,
-  50%,
-  70% {
-    transform: translate3d(-4px, -3px, 0);
-  }
-  40%,
-  60% {
-    transform: translate3d(4px, 3px, 0);
-  }
```

Comment:
I loved shake, but sometimes it's best to defer these kind of things to other developers.

Reply:
Shake can be used at a smaller scale since its such a big and rapid movement from a UX standpoint it's not the best. Shake can be put on the input for example since its small and minimal but I wouldn't put shake for correct answers as shake is more imposed for something wrong or to bring attention to something.